tests/runtest.proj

```diff
@@ -412,6 +412,21 @@ namespace $([System.String]::Copy($(Category)).Replace(".","_").Replace("\","").
 
   </Target>
 
+  <Target Name="RunPerfTests" Condition="'$(Performance)'=='true'">  
+    <Message Text="Executing steps for perf tests" Importance="High"/>  
+    
+    <!-- generate project.json for runtime dependency -->  
+    <MSBuild Projects="$(MSBuildProjectFile)"  
+             Targets="CreateTestRuntimeJsonFile"/>  
+     
+    <!-- Package the product -->  
+    <MSBuild Projects="$(MSBuildProjectFile)" Targets="CreateTestOverlay" />  
```

Comment:
Product actually should be coming from packages folder and not from this task. I think you can remove this. creation of correlation payload would work fine without this task. It happens in cloudtest.targets.